database/migrations/2025_04_30_211901_bump_version060600.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * SPDX-License-Identifier: MIT
+ * Copyright (c) 2017-2018 Tobias Reich
+ * Copyright (c) 2018-2025 LycheeOrg.
+ */
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Artisan;
+use Illuminate\Support\Facades\DB;
+use Symfony\Component\Console\Output\ConsoleOutput;
+use Symfony\Component\Console\Output\ConsoleSectionOutput;
+
+return new class() extends Migration {
+	private ConsoleOutput $output;
+	private ConsoleSectionOutput $msg_section;
+
+	public function __construct()
+	{
+		$this->output = new ConsoleOutput();
+		$this->msg_section = $this->output->section();
+	}
+
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up(): void
+	{
+		DB::table('configs')->where('key', 'version')->update(['value' => '060600']);
+		Artisan::call('cache:clear');
```

Comment:
The database migration script may not be the right place to call the command.

Reply:
The thing is we want to guarantee that this is run every version bump as the backend responses are likely to have changed. This literally made one of our release broken. See https://lycheeorg.dev/docs/releases.html#v651

Reply:
This is absolutely understandable for me I just wanted to bring in that technically it has nothing to do with the database migration and it might make sense to run a separate migration script that performs cleanups in the cache filesystem etc.


```php
<?php

define('PRODUCT_VERSION', '2.3.0');

if (version_compare(PRODUCT_VERSION, '3.0.0', '>=')) {
    // For version 3.0.0 or higher
    include 'features_v3.php';
} elseif (version_compare(PRODUCT_VERSION, '2.0.0', '>=')) {
    // For version 2.x
    include 'features_v2.php';
} else {
    // For older versions
    include 'features_legacy.php';
}
```